shadowproxy.go

```diff
@@ -3,15 +3,25 @@ package shadowproxy
 import (
 	"bytes"
 	"encoding/json"
+	"time"
 
 	"github.com/luraproject/lura/v2/config"
 )
 
-type ProxyConfig struct {
+type jsonConfig struct {
 	Host                     []string `json:"host"`
 	URLPattern               string   `json:"url_pattern"`
 	Method                   string   `json:"method"`
 	HostSanitizationDisabled bool     `json:"disable_host_sanitize"`
+	Timeout                  string   `json:"timeout"`
```

Comment:
Why do you need the timeout? Isn't the second request just discarded?

Reply:
yes, but maybe you would still want to specify a timeout for it, even if it's discarded, just to control for how long krakend should attempt to get a backend response. It also just feels consistent with the general krakend-config approach to me to be able to specify it...